azure-resources/DesktopVirtualization/hostPools/recommendations.yaml

```diff
@@ -1,102 +1,79 @@
   recommendationTypeId: null
   recommendationControl: Governance
   recommendationImpact: Medium
-  recommendationResourceType: Microsoft.DesktopVirtualization/hostPools
-  recommendationMetadataState: Active
-  longDescription: |
-    At least one Validation Pool to have early warning if a planned update to AVD causes an issue. support to adjust limits based on your business requirements. To handle a large number of users, consider scaling horizontally by creating multiple host pools.
-    Also check that the host pool has been used regularly to test planned updates.
-    Host pools are a collection of one or more identical virtual machines within Azure Virtual Desktop environment. We highly recommend you create a validation host pool where service updates are applied first. Validation host pools let you monitor service updates before the service applies them to your standard or non-validation environment. Without a validation host pool, you may not discover changes that introduce errors, which could result in downtime for users in your standard environment.
-    To ensure your apps work with the latest updates, the validation host pool should be as similar to host pools in your non-validation environment as possible. Users should connect as frequently to the validation host pool as they do to the standard host pool. If you have automated testing on your host pool, you should include automated testing on the validation host pool.
-  potentialBenefits: Early detection of update issues.
-  pgVerified: true
-  publishedToLearn: false
-  publishedToAdvisor: false
-  automationAvailable: no
-  tags: null
-  learnMoreLink:
-    - name: Learn More
-      url: "https://learn.microsoft.com/en-us/azure/virtual-desktop/configure-validation-environment?tabs=azure-portal"
-
-- description: Monitor Service Health and Resource Health of AVD
-  aprlGuid: a75a20e7-8cc0-4f7b-b4a9-e2476bd72429
-  recommendationTypeId: null
-  recommendationControl: Monitoring and Alerting
-  recommendationImpact: High
-  recommendationResourceType: Microsoft.DesktopVirtualization/hostPools
+  recommendationResourceType: n/a
   recommendationMetadataState: Active
   longDescription: |
-    Use Service Health to stay informed about the health of the Azure services and regions that you use to insure their availability.
-    Set up Service Health alerts so that you stay aware of service issues, planned maintenance, or other changes that might affect your Azure Virtual Desktop resources.
-    Use Resource Health to monitor your VMs and storage solutions.
-  potentialBenefits: Enhanced AVD uptime and awareness
+    Hybrid VMs should be in a unique OU. When using AD-joined, session hosts will benefit from using a unique OU to target specific AVD configurations per host pool. Examples include Fslogix, time out limits, session controls, and much more. It’s also important to segment Prod and DR organization units to ensure resources are configured per environment.
```

Comment:
Can you try to make this less than 300 characters? We initially didn't mess with adjusting the Specialized Workloads for the characters limits during the migration, but we'll slowly need to start transitioning these.